asterius/src/Asterius/Passes/DataOffsetTable.hs

```diff
@@ -35,7 +35,7 @@ makeDataOffsetTable :: AsteriusModule -> (SM.SymbolMap Word32, Word32)
 makeDataOffsetTable AsteriusModule {..} =
   swap $
     SM.mapAccum
-      ( \a ss -> (a + fromIntegral (fromIntegral (sizeofStatics ss) `roundup` 16), a)
+      ( \a ss -> (a + fromIntegral (fromIntegral (sizeofStatics ss) `roundup` 8), a)
```

Comment:
If #750 lands first this should be equivalent to setting [`segAlignment`](https://github.com/tweag/asterius/pull/750/files#diff-a18395ea112d86e48e2e7af6b3b5eac8R27-R30) to `8`.